src/rantanplan/core.py

```diff
@@ -214,7 +1519,7 @@ def get_syllables(word_list):
     return syllabified_words
 
 
-def get_scansion(text):
+def get_scansion(text, *_):
```

Comment:
This is not needed: `def get_scansion(text):``is just fine I believe.

Reply:
Oki, changed